docs/access_controls.md

```diff
@@ -0,0 +1,114 @@
+# CTMS Access Control Overview
+
+CTMS uses **Role-Based Access Control (RBAC)** to manage access to resources. The system consists of:
+- **Permissions**: Define specific actions that can be performed.
+- **Roles**: Group permissions together for easier management.
+- **API Clients**: The oAuth clients that authenticate and receive roles.
+
+---
+
+## Permissions
+
+**Permissions** define **what actions can be performed** within the system. They are the 
+**lowest level of access control** and must be assigned to **roles**.
+
+### Key Characteristics
+- Permissions are not assigned directly to API clients.
+- Roles are the only way permissions are granted to clients.
+- A single permission can be used in multiple roles.
+
+### Example Permissions
+| Permission Name | Description |
+|----------------|-------------|
+| `manage_contacts` | Grants the ability to create, edit, and delete contacts |
+| `view_updates` | Allows access to updated contacts |
```

Comment:
nit: I don't find `view_updates` very easy to grasp. Maybe `delete_contact` is better?

Reply:
Yep, hard to come up with arbitrary examples. :) I was trying to show various levels of CRUD operations that might come up.